contracts/libraries/Safe150Migration.sol

```diff
@@ -53,7 +52,7 @@ contract Safe150Migration is SafeStorage {
         address guard = getGuard();
```

Comment:
`checkGuard` isn't needed anymore - the guard no longer has a different interface ID right?

https://github.com/safe-global/safe-smart-account/blob/bf943f80fec5ac647159d26161446ac5d716a294/contracts/base/GuardManager.sol#L29

For reference, the migration script should be MUCH closer to https://github.com/safe-global/safe-smart-account/blob/main/contracts/libraries/Safe130To141Migration.sol

Reply:
Here: https://github.com/safe-global/safe-smart-account/pull/758/commits/80e02657edfcd65a032100314f0eb9fa73697b38

also moved event declaration to follow [style guide](https://docs.soliditylang.org/en/v0.8.20/style-guide.html#order-of-layout)